Modules/CodeTrek/Resources/views/sidebar.blade.php

```diff
@@ -2,12 +2,13 @@
     <h5 class="fw-bold border-bottom pb-2">CodeTrek Applicants</h5>
     <ul class="applicant-list list-unstyled">
         @foreach ($codeTrekApplicants as $codeTrekApplicant)
-        <li data-id="{{ $codeTrekApplicant->id }}" class="d-flex align-items-center">
-            <i class="fa fa-user mr-1"></i>
-            <h6 class="ml-1">
-                {{ $codeTrekApplicant->first_name }} {{ $codeTrekApplicant->last_name }}
-            </h6>
-        </li>
+            <li data-id="{{ $codeTrekApplicant->id }}" class="d-flex align-items-center">
+                <i class="fa fa-user mr-1"></i>
+                <a class="applicant-name" data-toggle="modal" data-target="#candidatefeedback{{ $codeTrekApplicant->id }}">
```

Comment:
Correct Naming Convention
```suggestion
                <a class="applicant-name" data-toggle="modal" data-target="#candidateFeedback{{ $codeTrekApplicant->id }}">
```